network_autogen/templates/GopherCAN_network.c.jinja2

```diff
@@ -22,6 +22,7 @@ CAN_INFO_STRUCT empty_param = {0};
         .TYPE = {{ param.type }},
         .SIZE = {{ param.type }}_SIZE,
         .ENC = {{ param.encoding }},
+        .ENC_SIZE = {{ param.length }},
```

Comment:
Just had a thought, what if we also gave an array in each group called .param_len that stores the encoding length of each parameter in each slot. The we could retain the functionality of being able to send parameters in different ways

Reply:
This is what I was suggesting earlier lol. If we want params to be in multiple groups though, we need a way to determine which one to send when you call send_parameter.

Reply:
I am an idiot it seems, I forgot about that whole conversation. This solution is good then